charts/sysdig-deploy/templates/NOTES.txt

```diff
@@ -10,6 +10,7 @@ Links for your deployment:
 Usage of global tags will override any local tags you might be using.
 {{- end }}
 
+{{- if .Values.agent.enabled }}
```

Comment:
WDYT to add this check in the same if condition in the line below, like:
```suggestion
{{- if (and .Values.agent.enabled (or .Values.agent.gke.autopilot .Values.global.gke.autopilot)) }}
```

Reply:
Hey @mavimo I did that before opening the PR but then it fails on this other two lines:
https://github.com/sysdiglabs/charts/blob/master/charts/sysdig-deploy/templates/NOTES.txt#L25
https://github.com/sysdiglabs/charts/blob/master/charts/sysdig-deploy/templates/NOTES.txt#L31
This is why I did `{{- if .Values.agent.enabled }}` for the entire block instead of adding `and .Values.agent.enabled` for each `if` entry